lib/log.js

```diff
@@ -66,7 +66,7 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function(event, uid, request) {
+Lug.prototype.activityEvent = function(event, uid, request, extraData) {
```

Comment:
We could consider just merging `uid` and `extraData` into a single object parameter here